app/controllers/application.js

```diff
@@ -62,6 +65,12 @@ export default Controller.extend(
 			});
 
 			this._super();
+
+			track({
+				action: trackActions.open,
+				category: 'connection-type',
+				label: `mobile-${this.get('connectionType')}`
```

Comment:
I don't think we need to have mobile- in label
we used to add mobile-wiki to every category